File: Python projects/Python Basics/Online_example_exams/exam_1/tennis_ranklist.py
from math import floor
from math import ceil

number_cups = int(input())
starting_points = int(input())
points = 0
number_wins = 0

for cups in range(number_cups):
    position = str(input())
    if position == 'W':
        points += 2000
        number_wins += 1
    elif position == 'F':
        points += 1200
    else:
        points += 720

print(f"Final points: {starting_points + points}")
print(f"Average points: {floor(points / number_cups)}")
print(f"{number_wins / number_cups * 100:.2f}%")

File: JavaScript projects/JavaScript Basics/For Loop/Exercises/tennis_ranklist.js
function tennisRanklist(input){
    let plays = Number(input[0]);
    let initPoints = Number(input[1]);
    let points = 0;
    let wins = 0;
    for (let i = 2; i < 2 + plays; i++){
        switch(input[i]){
            case "W":
                points += 2000;
                wins ++;
                break;
            case "F":
                points += 1200;
                break;
            case "SF":
                points += 720;
                break;
            default:
                break;
        }
    }
    console.log(`Final points: ${points + initPoints}`);
    console.log(`Average points: ${Math.floor(points / plays)}`);
    console.log(`${((wins / plays) * 100).toFixed(2)}%`);
}
tennisRanklist(["5","1400","F","SF","W","W","SF"]);